homeassistant/components/tplink/manifest.json

```diff
@@ -5,5 +5,55 @@
   "documentation": "https://www.home-assistant.io/integrations/tplink",
   "requirements": ["pyHS100==0.3.5.2"],
   "codeowners": ["@rytilahti", "@thegardenmonkey"],
-  "iot_class": "local_polling"
+  "iot_class": "local_polling",
+  "dhcp": [
+    {
+      "hostname": "hs*",
+      "macaddress": "1C3BF3*"
```

Comment:
KL130(EU), KP303(UK) are using this, but with different hostnames. My HS110(EU) uses unlisted, unknown vendor (based on macvendorlookup) B0:95:75.

Reply:
Tplink has an odd allocation strategy. They don't seem to constrain a specific device type to a specific range, and just use the next available for their device run.   I'll add yours.

Reply:
Looks like B0:95:75 is listed here at least https://maclookup.app/macaddress/B09575

Reply:
Oh, I just googled the first vendor lookup this time, which said unknown for that hs110 v4 plug... My older hs110 v1 plug has one that's already there in the PR (50:C7:BF) :)